### PR TITLE
Loginnav

### PR DESCRIPTION
**Pull request by Donnyd88**

### CODE REVIEW

File: src/component/welcome-logout/WelcomeLogout.jsx [hunk 1]
```diff
@@ -4,10 +4,11 @@ import styles from './WelcomeLogout.module.scss';
 
 const WelcomeLogout = ({ firstName, logout }) => {
     return (
-        <div className={styles.logoutContainer}>
-            <p className={styles.welcomeMessage}>Welcome, {firstName}!</p>
-            <button className={styles.logoutButton} onClick={logout}>Logout</button>
-        </div>
+        <>
+            <li className={styles.a}>Welcome, {firstName}!</li>
```
Comment (dinaolmelak) [inline src/component/welcome-logout/WelcomeLogout.jsx:8]: Considering we already have welcome on the `org-dashboard` already, can we  remove the `Welcome` here? OR we can add the first name next to the welcome found on the `org-dashboard` and then have this to just have the logout button.

**Review by dinaolmelak — CHANGES_REQUESTED** [review]
Thank you for removing the remanence of the old implementation of the logout. ✅
1. The `org-dashboard` logout does not work.
2. I realized that the picture is covering the nav bar on the `userpage`, I think we would need to bring the nav bar to the front and fix it to the top. I will work on that on #169 ticket.

File: .gitignore [hunk 2]
```diff
@@ -29,6 +29,7 @@ package-lock.json
 /backend/.env
 .env
 .env.production
+/backend/.env.production
```
Comment (dinaolmelak) [inline .gitignore:32]: There is no backend `.env.production` file

**Review by dinaolmelak — CHANGES_REQUESTED** [review]
Branch update needed

**Comment by dinaolmelak** [comment]
I saw that the functionality works for the `org-dashboard`, It should work for the `userpage` too but it is not working. I will review after the branch updates

**Comment by Donnyd88** [comment]
> I saw that the functionality works for the `org-dashboard`, It should work for the `userpage` too but it is not working. I will review after the branch updates

Do you want me to add the navbar to the userpage?

**Comment by dinaolmelak** [comment]
I believe userpage already has a navbar

**Comment by Donnyd88** [comment]
> I believe userpage already has a navbar

It did not. Or rather the NavBar component on the other pages that contained the welcome message and login button wasn't on that page. I just added it, though.

**Review by dinaolmelak — APPROVED** [review]

**Pull request merged**